cmd/list/firmware.go

```diff
@@ -45,3 +81,13 @@ var listFirmware = &cobra.Command{
 		table.Render()
 	},
 }
+
+func init() {
+	flagsDefinedListFirmware = &listFirmwareFlags{}
+
+	listFirmware.PersistentFlags().StringVar(&flagsDefinedListFirmware.server, "server", "", "server UUID")
+	listFirmware.PersistentFlags().StringVar(&flagsDefinedListFirmware.vendor, "vendor", "", "vendor name")
```

Comment:
We could support queries for multiple vendors or components too ("give me all supermicro or dell firmware for NICs").

Reply:
oh yea, ill look into how that would be done. gotta be a way to format the string input for the query to support that.

Reply:
While this is technically possible, its a bit out of scope, and for a proper implementation of it would require work to be done to serverservice to allow more complex queries.